java/client/src/org/openqa/selenium/support/ui/CheckedFunction.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
This is the wrong copyright notice. See CONTRIBUTING.md (or just copy from another java file, they've all been updated)